gnucash/register/register-gnome/combocell-gnome.c

```diff
@@ -606,56 +601,42 @@ gnc_combo_cell_modify_verify (BasicCell *_cell,
         return;
     }
 
-    //if (type_ahead_search)
-    {
-        // Type-ahead search, we match any substring of the account name.
-        full_store = cell->shared_store_full;
-        match_str = gnc_combo_cell_type_ahead_search(newval, full_store, box);
-        ss = newval_chars;
-        es = -1;
-        cp = newval_chars;
-        // Do not change the string in the type-in box.
-        box_string = newval;
-    }
-    //else
-    {
-        QuickFill *match = NULL;
-        // Match at start of account name using the quickfill
-        match = gnc_quickfill_get_string_match (box->qf, newval);
-        match_str2 = g_strdup(gnc_quickfill_string (match));
-        ss2 = newval_chars;
-        es2 = -1;
-        cp2 += change_chars;
-        // Update the string in the type-in box with the matched string.
-        box_string2 = match_str2;
-    }
-    
-    if(match_str2 != NULL)
+    // Try the start-of-name match using the quickfill
+    QuickFill *match = NULL;
```

Comment:
C++ must be deeply ingrained in your fingers, you keep getting bitten by declaring variables in the middle. Did it compile without error on Xcode?

Reply:
Ah crap, sorry about that. Yes, it compiles fine on Xcode. The check in is temporary of course, since i will remove the preference when I'm done, but I can't believe I let this one slip through. Do you know if there's a way for travis to alert you if your build fails?

Reply:
I think you need to set up travis on your own repo to get notifications. I wonder if Xcode is compiling it as c++ because of something internal or because cmake is telling it to.